docarray/index/backends/in_memory.py

```diff
@@ -442,9 +444,16 @@ def __contains__(self, item: BaseDoc):
                 f"item must be an instance of BaseDoc or its subclass, not '{type(item).__name__}'"
             )
 
-    def persist(self, file: str = 'in_memory_index.bin') -> None:
+    def persist(self, file: Optional[str] = None) -> None:
         """Persist InMemoryExactNNIndex into a binary file."""
-        self._docs.save_binary(file=file)
+        DEFAULT_INDEX_FILE_PATH = 'in_memory_index.bin'
```

Comment:
I'd assign this inside init as a default value

Reply:
I do not want, because I do not want it to have a default value at `__init__`.  This is just for backwards compatibility at `persist` time

Reply:
ok but why wouldn't it be backward compatible?

Reply:
I want it to be only at `persist level`. If I put at top level, it seems to apply at load time as well